.swiftlint.yml

```diff
@@ -82,7 +82,7 @@ file_header:
   required_pattern: |
                     \/\/(
                     \/\/ (SwiftGen(Kit)?|.* UnitTests)
-                    \/\/ Copyright © 2019 SwiftGen
+                    \/\/ Copyright © 2020 SwiftGen
```

Comment:
We need to update `SwiftGen.xcodeproj/xcshareddata/IDETemplateMacros.plist` accordingly too